zuul-core/src/main/java/com/netflix/netty/common/throttle/RejectionUtils.java

```diff
@@ -49,7 +49,7 @@
 
     // TODO(carl-mastrangelo): add tests for this.
 
-    private static final HttpResponseStatus REJECT_CLOSING_STATUS = new HttpResponseStatus(999, "Closing(Rejection)");
+    public static final HttpResponseStatus REJECT_CLOSING_STATUS = new HttpResponseStatus(999, "Closing(Rejection)");
```

Comment:
This needs to be public for the rejection handler to correctly set the http code to -1.   The response status cant be -1 when passed to the the ctor of this class, so I need to check it at the rejection handler.